tools/contrib/junit_xml/__init__.py

```diff
@@ -0,0 +1,322 @@
+#!/usr/bin/env python
```

Comment:
That would work directly against one of the goals here, to reduce the number of confusing little subprocesses and files that serve only to indirect and make it harder to see how information is moving through the scripts.  So I don't want to do that.  Perhaps some others may have opinions or suggestions.